src/BugsnagUnity/TimingTrackerBehaviour.cs

```diff
@@ -14,6 +14,7 @@ private void Awake()
         {
             // Make sure that the tracker persists accross scenes.
             DontDestroyOnLoad(gameObject);
+            gameObject.hideFlags = HideFlags.DontSave;
```

Comment:
Is it better to use [`HideFlags.DontSave`](https://docs.unity3d.com/ScriptReference/HideFlags.DontSave.html) or [`HideFlags.HideAndDontSave`](https://docs.unity3d.com/ScriptReference/HideFlags.HideAndDontSave.html) ?

Reply:
I don't want to hide the object because it makes troubleshooting potential future issues much harder both for users and for us.